pkg/storagev2/internal/internal.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package internal
+
+import (
+	"context"
+
+	"github.com/ethersphere/bee/pkg/storagev2"
+)
+
+// Storage groups the storage.Store and storage.ChunkStore interfaces with context..
+type Storage interface {
+	Ctx() context.Context
+	Storage() storage.Store
```

Comment:
shouldn't this be called `Store() storage.Store`?

Reply:
Done.